portfolio/src/main/webapp/script.js

```diff
@@ -42,10 +42,53 @@ function fadeOut(){
 }
 
 //From Week-3 Tutorial
-function getGreeting() {
-  console.log('Handling the response.');
-  fetch('/data').then(response => response.text()).then((quote) => {
-    document.getElementsByClassName('greeting-cont')[0].innerText = quote;
+function getComments() {
+  numComments = localStorage.getItem("num-of-comments");
+  console.log(numComments);
+  fetch(`/comments?num-comments=${numComments}`).then(response => response.json()).then((comment) => {
+    console.log(comment);
+    container = document.getElementsByClassName('comments-section')[0];
+    container.innerHTML = "";
+    for (item in comment){
```

Comment:
`for (const c of comments) {`

add space after closing `)`

Reply:
done

Reply:
let's use "for...of" instead of "for...in", since "for...of" handles iterable objects explicitly (which is what we want from an iterable list), as opposed to enumerating the properties, which has different semantics.

Reply:
done